app/seller/seller-form.tsx

```diff
@@ -0,0 +1,317 @@
+            console.log("Error: ", error);
+        }
+    };
+
+    const tags = PostTagsEnum.options.map((tag) => ({
+        label: tag,
+        value: tag,
+    }));
+
+    return (
+        <>
+            <View className="p-4">
+                <Text className="mb-1 font-bold">Caption</Text>
+                <Controller
+                    control={control}
+                    name="caption"
+                    render={({ field: { onChange, onBlur, value } }) => (
+                        <TextInput
+                            className="rounded-md bg-gray-300 px-3 py-1.5 text-base text-gray-900"
+                            placeholder="Enter Caption"
+                            onBlur={onBlur}
+                            onChangeText={onChange}
+                            value={value}
+                        />
+                    )}
+                />
+                {errors.caption && (
+                    <Text className="mt-1 text-red-500">
+                        {errors.caption.message}
+                    </Text>
+                )}
+                <Text className="mt-2 font-bold">Tags</Text>
+                <Controller
+                    control={control}
+                    name="tags"
+                    render={({ field: { onChange, value } }) => (
+                        <MultiSelect
+                            style={{
+                                backgroundColor: "rgb(209, 213, 219)",
+                                borderRadius: 6,
+                                padding: 8,
+                                marginTop: 4,
+                            }}
+                            selectedTextStyle={{ color: "black" }}
+                            data={tags}
+                            labelField="label"
+                            valueField="value"
+                            placeholder="Select tags"
+                            value={value}
+                            onChange={(items) => {
+                                onChange(items);
+                            }}
+                            selectedStyle={{
+                                borderRadius: 12,
+                                backgroundColor: "skyblue",
+                            }}
+                            activeColor="skyblue"
+                        />
+                    )}
+                />
+                {errors.tags && (
+                    <Text className="mt-1 text-red-500">
+                        {errors.tags.message}
+                    </Text>
+                )}
+            </View>
+            <ScrollView>
+                <View className="p-4">
+                    <View>
+                        {errors.tags && (
+                            <Text className="mt-1 text-red-500">
+                                {errors.tags.message}
+                            </Text>
+                        )}
+                    </View>
+                    {fields.map((field, index) => (
```

Comment:
this is not needed (but will greatly improve the readability of this component), but its nice if you can extract the component here in a new file, maybe `ItemsForm.tsx`? just send the needed data through props (which is a hassle i know but i really think its worth it).